source/module.c

```diff
@@ -1000,6 +1003,9 @@ PyInit__cbor2(void)
 
 #undef INTERN_STRING
 
+    if (!_CBOR2_date_ordinal_offset &&
+            !(_CBOR2_date_ordinal_offset = PyLong_FromLong(719163)))
```

Comment:
Would be helpful (to me, who doesn't know CBOR expect as a user) to know what this magic number means.

Reply:
See the CBOR definition for [semantic tag 100](https://datatracker.ietf.org/doc/html/draft-ietf-cbor-date-tag-04#section-2.1). 719163 is the number of days between the minimum supported CBOR date and the [minimum supported Python date ordinal](https://docs.python.org/3/library/datetime.html#datetime.date.fromordinal) (which starts from January 1 of year 1).